frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/HepatitisCAoeForm.tsx

```diff
@@ -0,0 +1,141 @@
+
+  const {
+    showSymptomsError,
+    showSymptomOnsetDateError,
+    hasSymptoms,
+    showSymptomSelectionError,
+    symptoms,
+  } = generateSymptomAoeConstants(
+    responses,
+    hasAttemptedSubmit,
+    hepatitisCSymptomDefinitions
+  );
+
+  const CHECKBOX_COLS_TO_DISPLAY = 3;
+  return (
+    <div
+      className="grid-col"
+      id={`hepatitis-c-aoe-form-${testOrder.patient.internalId}`}
+    >
+      <div className="grid-row">
+        <div className="grid-col-auto">
+          <GenderOfSexualPartnersAoe
+            testOrderId={testOrder.internalId}
+            responses={responses}
+            hasAttemptedSubmit={hasAttemptedSubmit}
+            onResponseChange={onResponseChange}
+            required
+          />
+        </div>
+      </div>
+      <div className="grid-row">
+        <div className="grid-col-auto">
+          <PregnancyAoe
+            testOrderId={testOrder.internalId}
+            responses={responses}
+            hasAttemptedSubmit={hasAttemptedSubmit}
+            onResponseChange={onResponseChange}
+            required
+          />
+        </div>
+      </div>
+      <div className="grid-row">
+        <div className="grid-col-auto">
+          <YesNoRadioGroup
+            name={`has-any-symptoms-${testOrder.internalId}`}
+            legend="Is the patient currently experiencing or showing signs of symptoms?"
+            value={hasSymptoms}
+            onChange={onHasAnySymptomsChange}
+            required
+            validationStatus={showSymptomsError ? "error" : undefined}
+            errorMessage={
+              showSymptomsError && "Please answer this required question."
+            }
+          />
+        </div>
+      </div>
+      {hasSymptoms === "YES" && (
+        <div className={"grid-row grid-gap width-full"}>
+          <div className={"grid-col-auto"}>
+            <Checkboxes
+              boxes={hepatitisCSymptomDefinitions.map(({ label, value }) => ({
+                label,
+                value,
+                checked: symptoms[value],
+              }))}
+              legend="Select any symptoms the patient is experiencing"
+              name={`symptoms-${testOrder.internalId}`}
+              onChange={(e) => onSymptomsChange(e, symptoms)}
+              numColumnsToDisplay={CHECKBOX_COLS_TO_DISPLAY}
+              validationStatus={showSymptomSelectionError ? "error" : undefined}
+              errorMessage={
+                showSymptomSelectionError
+                  ? "Please answer this required question."
+                  : ""
+              }
+            />
+          </div>
+          <div className="grid-col-auto">
+            <TextInput
+              data-testid="symptom-date"
+              name={`symptom-date-${testOrder.internalId}`}
+              type="date"
+              label="Date of symptom onset"
+              aria-label="Symptom onset date"
+              className={""}
+              min={formatDate(new Date("Jan 1, 2020"))}
```

Comment:
(non-blocking) After looking at this again, I'm remembering we initially set this minimum date because this was for Covid. I'm wondering if we need to adjust this for the new forms/diseases? 🤔

Reply:
Really good point for non-COVID diseases! Let's ask in sync today whether we should update this